aptos-move/parallel-executor/src/proptest_types/tests.rs

```diff
@@ -80,16 +93,67 @@ proptest! {
         abort_transactions in vec(any::<Index>(), 5),
         skip_rest_transactions in vec(any::<Index>(), 5),
     ) {
-        prop_assert!(run_transactions(universe, transaction_gen, abort_transactions, skip_rest_transactions));
+        prop_assert!(run_transactions(universe, transaction_gen, abort_transactions, skip_rest_transactions, 1));
     }
 
     #[test]
-    fn imprecise_read_estimation(
+    fn dynamic_read_writes_mixed(
         universe in vec(any::<[u8; 32]>(), 100),
-        transaction_gen in vec(any::<TransactionGen<[u8;32]>>(), 3000).no_shrink(),
-        abort_transactions in vec(any::<Index>(), 5),
-        skip_rest_transactions in vec(any::<Index>(), 5),
+        transaction_gen in vec(any_with::<TransactionGen<[u8;32]>>(TransactionGenParams::new_dynamic()), 3000).no_shrink(),
+        abort_transactions in vec(any::<Index>(), 3),
```

Comment:
why 3 instead of 5?

Reply:
5 will actually run less transactions on average because it will abort earlier - so since I used 3000 instead of 5000 txns for this test, decided to do 3 instead of 5 of them being abort.